app/assets/stylesheets/modules/spotlight_overrides.scss

```diff
@@ -220,3 +220,9 @@ h5,
     z-index: 1;
   }
 }
+
+// This allows for the skip links to appear above the header
+// instead of overlapping with the header. 
+#skip-link {
```

Comment:
Should this change be in the component library?

Reply:
We can put it in there to ensure this overrides any absolute positioning, since that does seem to interfere.